iotdb/src/main/java/org/apache/iotdb/db/metadata/MTree.java

```diff
@@ -212,6 +212,34 @@ public void setStorageGroup(String path) throws PathErrorException {
     setDataFileName(path, cur);
   }
 
+  /**
+   * check whether the input path is storage group or not
+   * @param path input path
+   * @return if it is storage group, return true. Else return false
+   */
+  public boolean checkStorageGroup(String path) {
+    String[] nodeNames = path.split(DOUB_SEPARATOR);
+    MNode cur = root;
+    if (nodeNames.length <= 1 || !nodeNames[0].equals(root.getName())) {
+      return false;
+    }
+    int i = 1;
+    while (i < nodeNames.length - 1) {
+      MNode temp = cur.getChild(nodeNames[i]);
+      if (temp == null && temp.isStorageLevel()) {
```

Comment:
if temp is null how to invoke isStorageLevel() function?

Reply:
should be "or", fixed